### PR TITLE
Build out sorting functionality feature to conformance page

### PR DESCRIPTION
**Pull request by nekevss**
Related to #148

Currently makes adjustments to allow sorting alphabetically and reverse alpha.

There's probably a little more styling needed on the mobile side of things, but I thought I'd throw this up early for feedback. :smile:

### CODE REVIEW

**Comment by nekevss** [comment]
There are now a whole host of sorting options :smile:

**Comment by jedel1043** [comment]
Was testing the functionality and I found some bugs:
- Some ordering options don't work:
image
image
- The selected sort option is not preserved after clicking a suite or the breadcrumbs.

However, I think we can merge this as it is, and open some separate issues for those.

**Review by jedel1043 — APPROVED** [review]

File: src/components/conformance/utils.ts [hunk 1]
```diff
@@ -15,17 +16,106 @@ export function createState(
...
+  },
+  {
+    id: "most-ignored",
+    name: "Most Ignored",
+    callback: (a, b) => b.stats.ignored - a.stats.ignored,
+  },
+  {
+    id: "least-ignored",
+    name: "Least Ignored",
+    callback: (a, b) => a.stats.passed - b.stats.passed,
```
Comment (nekevss) [inline src/components/conformance/utils.ts]: This is where that bug comes from for `Least Ignored` (I forgot to switch over the property field to ignore)

```suggestion
    id: "least-ignored",
    name: "Least Ignored",
    callback: (a, b) => a.stats.ignored - b.stats.ignored,
```

**Comment by nekevss** [comment]
This is going to need a rebase after #155 for the fix, and the behavior should be easy to fix for the above.

**Review by HalidOdat — APPROVED** [review]
Nice work! :)

**Pull request merged**